src/app/static/src/tests/components/runWorkflow/runWorkflow.test.ts

```diff
@@ -15,13 +16,15 @@ describe(`runWorkflow`, () => {
     ]
 
     const workflowMetadata = [{
+        name: "interim report",
```

Comment:
At least some of these runWorflow tests are failing because this `workflowMetadata` is an array rather than an single object. Take out the surrounding `[`..`]` from this `workflowMetadata` and the helper method will then succeed.